src/vm/dllimport.cpp

```diff
@@ -6362,6 +6371,55 @@ NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryModuleViaHost(NDirectMethodDesc * pMD,
     return (NATIVE_LIBRARY_HANDLE)hmod;
 }
 
+NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryModuleViaCallBack(NDirectMethodDesc * pMD, LPCWSTR wszLibName)
+{
+#ifndef PLATFORM_UNIX
+    if (IsWindowsAPI(wszLibName))
```

Comment:
What is the rationale behind blocking this?

Reply:
My only concern with not blocking this is the scenario where we deploy Windows API sets on downlevel Windows SKUs  (i.e. Windows 7). The issue here would be why are we looking in the system if they are app local? This is largely a hedge about not trusting Windows to deploy API sets that are guaranteed backwards compatible. I think that is a fair assumption the API sets will always be backwards compat, but note that the behavior will be different on different platforms if we rely on a system API set in some cases and our applocal version of the API sets in others.